python_modules/dagster/dagster/core/definitions/decorators/op.py

```diff
@@ -1,5 +1,5 @@
 from functools import update_wrapper
-from typing import Any, Callable, Dict, List, Optional, Sequence, Set, Union
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Sequence, Set, Union
```

Comment:
Oops, I'm going to move these changes that don't have anything to do with asset defs to a separate PR.